src/types/ServerError.js

```diff
@@ -0,0 +1,12 @@
+const ServerError = {
```

Comment:
```suggestion
const serverError = {
```
Incorrect naming - uppercased letter is reserved for classes, typescript types, react components etc.

Reply:
I did it, because my "ServerError" it's my type of enum:)

Reply:
> I did it, because my "ServerError" it's my type of enum:)

There are no TS here, so I don't believe you)